### PR TITLE
Update package to moden Julia Project.toml format

### PR DESCRIPTION
**Pull request by staticfloat**

### CODE REVIEW

**Comment by staticfloat** [comment]
@YingboMa

File: Manifest.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This file is machine-generated - editing it directly is not advised
+
+julia_version = "1.10.0-DEV"
```
Comment (YingboMa) [inline Manifest.toml]: Do we need `Manifest.toml` here?

Comment (staticfloat) [reply Manifest.toml]: No, there's no real need for it.

Comment (YingboMa) [reply Manifest.toml]: Should we remove it?

**Review by YingboMa — APPROVED** [review]

**Pull request merged**